electron/.gitignore

```diff
@@ -1,2 +1,2 @@
 dist/
-node-modules/
\ No newline at end of file
+node_modules/
```

Comment:
This one should be removed as it's already in the root `.gitignore`, so changing it to the correct one has no effect. I think it's better to just remove it.

Reply:
Fixed